### PR TITLE
Add README for `efo-mondo-map.tsv`

### PR DESCRIPTION
**Pull request by sangeetashukla**
### Purpose/implementation Section
Add a README for the module

#### What GitHub issue does your pull request address?
[Issue 182](https://github.com/PediatricOpenTargets/ticket-tracker/issues/182)

### Directions for reviewers. Tell potential reviewers what kind of feedback you are soliciting.
Please read through the description for EFO and MONDO codes and share feedback.

#### Is the analysis in a mature enough form that the resulting figure(s) and/or table(s) are ready for review?
Yes

#### Reproducibility Checklist

- [ ] The dependencies required to run the code in this pull request have been added to the project Dockerfile.
- [ ] This analysis has been added to continuous integration.

#### Documentation Checklist

- [Y] This analysis module has a `README` and it is up to date.
- [ ] This analysis is recorded in the table in `analyses/README.md` and the entry is up to date.
- [ ] The analytical code is documented and contains comments.

### CODE REVIEW

File: analyses/efo-mondo-mapping/README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+##EFO and MONDO mapping
+
+The Experimental Factor Ontology (EFO) is a resource that provides systematic description of experimental variables including cancer groups available in EBI databases and some other external projects. The scope of EFO is to support the annotation, analysis and visualization of data handled by many groups at the EBI and as the core ontology for OpenTargets.org
```
Comment (jharenza) [inline analyses/efo-mondo-mapping/README.md]:
```suggestion
The Experimental Factor Ontology (EFO) is a resource that provides systematic description of experimental variables including cancer groups available in EBI databases and some other external projects. The scope of EFO is to support the annotation, analysis and visualization of data handled by many groups at the EBI and as the core ontology for OpenTargets.org
```

Can you add what EBI is here? Also, will you add a hyperlink for OpenTargets.org and update this to be one sentence per line?

**Review by runjin326 — APPROVED** [review]
Thanks @sangeetashukla for adding the description for the 4 cancer groups that are not included now. Looks good to me!

File: analyses/efo-mondo-mapping/README.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+The Mondo Disease Ontology (Mondo) is another independent resource aiming to harmonize disease definitions. 
+
+The purpose of maintaining this dataset is to map each cancer group found in the histologies.tsv to its appropriate EFO and MONDO codes.
+
+###Format
+
+The EFO codes may have prefix “EFO_”, “MONDO_” or “Orphanet_”, while the MONDO codes have a prefix “MONDO_”; each followed by seven digits.
+
+As more cancer groups or subtypes are added to existing histologies dataset and/or in case of any ambiguity, these codes will be revised.
+Currently, all non-NA cancer groups from the histologies files except 'Epilepsy', 'Dysplasia/Gliosis;Glial-neuronal tumor NOS', 'Arteriovenous malformation', and 'Reactive connective tissue' are included in this file.
```
Comment (jharenza) [inline analyses/efo-mondo-mapping/README.md]:
```suggestion
Currently, all non-NA cancer groups from `histologies.tsv` except 'Epilepsy', 'Dysplasia/Gliosis;Glial-neuronal tumor NOS', 'Arteriovenous malformation', and 'Reactive connective tissue' are included `efo-mondo-map.tsv`.
```

File: analyses/efo-mondo-mapping/README.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+##EFO and MONDO mapping
+
+The Experimental Factor Ontology (EFO) is a resource that provides systematic description of experimental variables including cancer groups available in EBI databases and some other external projects. The scope of EFO is to support the annotation, analysis and visualization of data handled by many groups at the EBI and as the core ontology for OpenTargets.org
+
+
+The Mondo Disease Ontology (Mondo) is another independent resource aiming to harmonize disease definitions. 
+
+The purpose of maintaining this dataset is to map each cancer group found in the histologies.tsv to its appropriate EFO and MONDO codes.
```
Comment (jharenza) [inline analyses/efo-mondo-mapping/README.md]:
```suggestion
The purpose of maintaining `efo-mondo-map.tsv` is to map each cancer group found in the `histologies.tsv` to its appropriate EFO and MONDO codes.
```

**Review by jharenza — CHANGES_REQUESTED** [review]
thanks @sangeetashukla - just a few quick change requests!

File: analyses/efo-mondo-mapping/README.md (hunk 1 above)
Comment (sangeetashukla) [reply analyses/efo-mondo-mapping/README.md]: Done. Please take another look and let me know how you like it.

File: analyses/efo-mondo-mapping/README.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
+# EFO and MONDO mapping
+
+
+## Introduction and Purpose
+
+The Experimental Factor Ontology (EFO) is a resource that provides systematic description of experimental variables including cancer groups available in [European Bioinformatics Institute](https://www.ebi.ac.uk/ "EMBL-EBI's Homepage") databases and some other external projects.
```
Comment (jharenza) [inline analyses/efo-mondo-mapping/README.md]:
```suggestion
The Experimental Factor Ontology (EFO) is a resource that provides systematic description of experimental variables including cancer groups available in [European Bioinformatics Institute](https://www.ebi.ac.uk/) databases and some other external projects. 
```

**Review by jharenza — COMMENTED** [review]
just one minor update on the link!

**Comment by sangeetashukla** [comment]
> just one minor update on the link!

@jharenza 
Done.

Let me know if we need more changes to this PR, and once it is merged I will delete the local branch.

File: analyses/efo-mondo-mapping/README.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
+# EFO and MONDO mapping
+
+
+## Introduction and Purpose
+
+The Experimental Factor Ontology (EFO) is a resource that provides systematic description of experimental variables including cancer groups available in [European Bioinformatics Institute](https://www.ebi.ac.uk/) databases and some other external projects. 
+The scope of EFO is to support the annotation, analysis and visualization of data handled by many groups at the EBI and as the core ontology for [OpenTargets](https://www.opentargets.org/ "OpenTargets' Homepage").
```
Comment (jharenza) [inline analyses/efo-mondo-mapping/README.md]:
```suggestion
The scope of EFO is to support the annotation, analysis and visualization of data handled by many groups at the EBI and as the core ontology for [OpenTargets](https://www.opentargets.org/).
```
This doesn't seem to have gone through - I just updated what it should look like, though. Here is a section on [links](https://docs.github.com/en/github/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#links).

Comment (sangeetashukla) [reply analyses/efo-mondo-mapping/README.md]: @jharenza That's weird. I pushed this change last week. It looks fine in my branch. 
Anyway, thank you.

**Review by jharenza — APPROVED** [review]
thanks!

**Pull request merged**